README-FIRST.txt

```diff
@@ -0,0 +1,6 @@
+Please see the documents directory "docs" and read through the file
```

Comment:
Note that README-FIRST-RAMS is moved to the docs directory and this file is added in its place.